include/swift/AST/Decl.h

```diff
@@ -927,6 +927,15 @@ class alignas(1 << DeclAlignInBits) Decl {
   /// @_originalDefinedIn attribute, this function returns this module name.
   StringRef getAlternateModuleName() const;
 
+  // Is this Decl an SPI? It can be marked with @_spi or it can be inherited.
```

Comment:
Nit: IMO "it can be inherited" could be misinterpreted as talking about class inheritance. How about "It can be directly marked `@_spi` or is defined in an `@_spi` context"?